test/TestConfig/scripts/tools/getDependencies.pl

```diff
@@ -66,9 +66,9 @@
 # Define a a hash for each dependent jar
 # Contents in the hash should be: url => , fname =>, sha1 =>
 my %asm_all = (
-	url => 'http://central.maven.org/maven2/org/ow2/asm/asm-all/6.0_BETA/asm-all-6.0_BETA.jar',
+	url => 'https://repository.ow2.org/nexus/content/repositories/snapshots/org/ow2/asm/asm/7.0-beta-SNAPSHOT/asm-7.0-beta-20180911.182645-11.jar',
```

Comment:
@smlambert @llxia we had to upgrade ASM to a newer level for the condy bytecode support. I'm just verifying this change is ok from the test perspective. Please let us know. Thanks!

Reply:
LGTM. 
- https://ci.eclipse.org/openj9/job/test.getDependency/ - may need to manually launch after  merge. It is schedule to run every hour. If any test build is launched between the merge and new test.getDependency build, the test build will fail. 
- Same for https://ci.adoptopenjdk.net/view/all/job/test.getDependency/ and internal Jenkins